packages/in_app_purchase/lib/src/in_app_purchase/purchase_details.dart

```diff
@@ -72,24 +70,6 @@ enum PurchaseStatus {
   error
 }
 
-/// Error of a purchase process.
-///
-/// The error can happen during the purchase, or restoring a purchase.
-/// Errors from restoring a purchase are not indicative of any errors during the original purchase.
-class PurchaseError {
```

Comment:
Could we change these in a follow up PR? This does make sense but I think it would be good to get the breaking change and the bug fix separated from each other.

Reply:
Done. I still moved the `PurchaseError` and `PurchaseSource` to the in_app_purchase_connect file so both product and purchase and use this.